org.eclipse.m2e.core/src/org/eclipse/m2e/core/internal/M2EUtils.java

```diff
@@ -184,4 +189,38 @@ public static void copyProperties(Properties to, Properties from) {
       }
     }
   }
+
```

Comment:
For Hash operations I always found commons.DigestUtils quite handy...

Reply:
Good suggestion!
The library `apache.commons.codec` is much smaller and used already in M2E and other Eclipse plug-ins so I think it is a good trade. Thanks.